packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/meshes/meshPropertyGridComponent.tsx

```diff
@@ -166,11 +166,12 @@ export class MeshPropertyGridComponent extends React.Component<
             mesh.reservedDataStore.originalMaterial = null;
             this.setState({ displayNormals: false });
         } else {
-            if (!(BABYLON as any).NormalMaterial) {
-                this.setState({ displayNormals: true });
-                Tools.LoadScript("https://preview.babylonjs.com/materialsLibrary/babylonjs.materials.js", () => {
-                    this.displayNormals();
-                });
+            if (!NormalMaterial) {
+                // this.setState({ displayNormals: true });

```

Comment:
Can't this be removed instead of just commented?

Reply:
I added a warning instead of logging. It is needed, as it is possible that the inspector will be used without loading the needed libs